src/middleware/auth.js

```diff
@@ -72,3 +78,25 @@ function validateTokenType(type) {
 
   return true
 }
+
+function tokenExpiryCheck(decodedToken, res) {
```

Comment:
This function can be replaced/use the `jsonwebtoken.verify` method. If the token is expired, the `verify` method will throw an error (which you can catch). This would be preferable to having our own method of checking if a token is expired or not as we will not need to maintain it.